src/coreclr/src/inc/crosscomp.h

```diff
@@ -113,8 +113,7 @@ typedef struct DECLSPEC_ALIGN(8) _T_CONTEXT {
 // each frame function.
 //
 
-#ifndef TARGET_UNIX
-#ifdef HOST_X86
+#if defined(HOST_WINDOWS) && defined(HOST_X86)
```

Comment:
This is not immediately obvious why this is a `ARM` fix.  This code is inside a 
```C++
#if !defined(HOST_ARM) && defined(TARGET_ARM) // Non-ARM Host managing ARM related code
```
block.

We are cross compiling for ARM on a X86 host.  We need these defines.

Reply:
Jan Kotas has been asking me to change !HOST_UNIX to HOST_WINDOWS, so I just did it here too.

Reply:
Is this really `HOST_WINDOWS`, or is it `TARGET_WINDOWS`. Before it only code for windows had it, now its things that run on windows.

Reply:
Ok